wedlocks/package.json

```diff
@@ -43,7 +32,7 @@
   "scripts": {
     "build": "netlify build",
     "dev": "UNLOCK_ENV=dev netlify functions:serve",
-    "test": "UNLOCK_ENV=test jest",
+    "test": "UNLOCK_ENV=test vitest run --no-threads --coverage --environment=node",
```

Comment:
You can remove --no-threads flag. I added it in locksmith because our tests weren't written correctly for parallelism. I think it would work here. If not, add it back.